### PR TITLE
[vue]: clean code

### PR DESCRIPTION
**Pull request by qmonmert**
Vuejs: clean code (unused imports & methods, and bad formatting code)

---

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by avdev4j** [comment]
hi @qmonmert 
could you please add [vue] in your commit message to avoid triggering Angular and react builds?

**Comment by qmonmert** [comment]
@avdev4j ok sorry

**Comment by avdev4j** [comment]
hum, [vue] seems not working :(

File: generators/client/templates/vue/src/main/webapp/app/account/activate/activate.vue.ejs [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-              <span v-html="$t('activate.messages.success')"><strong>Your user account has been activated.</strong> Please </span>
-              <a class="alert-link" v-on:click="openLogin" v-text="$t('global.messages.info.authenticated.link')">sign in</a>.
-          </div>
-          <div class="alert alert-danger" v-if="error" v-html="$t('activate.messages.error')">
-              <strong>Your user could not be activated.</strong> Please use the registration form to sign up.
-          </div>
-      </div>
-  </div>
-</div>
+    <div>
```
Comment (pascalgrimaud) [inline generators/client/templates/vue/src/main/webapp/app/account/activate/activate.vue.ejs:2]: here we're using 4 spaces for indent, and in error.vue.ejs we're using 2 spaces. I think we should be consistent

So maybe we can use prettier for formatting our vue file ? Could we try it ?

Comment (qmonmert) [reply generators/client/templates/vue/src/main/webapp/app/account/activate/activate.vue.ejs:2]: @pascalgrimaud I think that we could try in an other PR :)

Comment (qmonmert) [reply generators/client/templates/vue/src/main/webapp/app/account/activate/activate.vue.ejs:2]: @pascalgrimaud I try in this PR ;)

Comment (mshima) [reply generators/client/templates/vue/src/main/webapp/app/account/activate/activate.vue.ejs:2]: To use prettier we have to rename from `.ejs` to `.vue` if there is no ejs template.

**Comment by mshima** [comment]
> hum, [vue] seems not working :(

Working at PR title.

**Comment by pascalgrimaud** [comment]
FYI, I resolved the conflict in your branch @qmonmert 
Thanks for the work !

**Comment by qmonmert** [comment]
thanks @pascalgrimaud but why jobs are pending ?

**Comment by SudharakaP** [comment]
> hum, [vue] seems not working :(

@avdev4j @pascalgrimaud : It seems that 4 days ago GitHub (maybe silently) removed the head_commit from the context element; https://github.community/t/accessing-commit-message-in-pull-request-event/17158/5. I don't find official notice of this; but if so other tags such as `[skip ci]` will also have problems. We will need to find a different workaround for all our repos probably. :cry:

**Pull request merged**